Bug met JIRA en SP dashboard

Log regel
> "message":"Unable to create the Jira issue.","context":["JSON property \"expand\" in class \"JiraRestApi\\Issue\\IssueSearchResult\" must not be NULL"],"level":500,"level_name":"CRITICAL"


Andere belangrijke info
> Er is afgelopen woensdag een update van JIRA geweest. Er zou geen aanpassingen geweest moeten zijn aan de API. Er staat een vraag uit, maar daarover weten wij op dit moment niet meer dan dit.


Er zijn verschillende "verzoeken":
- Productieverzoeken leiden tot een foutmelding in de UI (change komt wel in Manage terecht, maar geen JIRA ticket).
- Change requests aan een productiekoppelingen leiden ook tot dezelfde foutmelding (in UI). In Manage komt geen change binnen, en geen ticket in JIRA. Deze foutmelding:
 ![Image](https://github.com/user-attachments/assets/2ea16e03-f71c-4917-974c-b28cfba85069)

- Connection request en delete request lijken wel goed te gaan (komen wel in Manage, krijgt wel een ticket in JIRA).

Het lijkt nu sterk samen te hangen met
[2025-05-14T16:01:20] [ERROR] (unknown)
>   ➤ Message : {"errorMessages":[],"errors":{"fixVersions":"Field 'fixVersions' cannot be set. It is not on the appropriate screen, or unknown."}}" at [JiraClient.ph](http://jiraclient.ph/)p line 286","context":{"exception":{"class":"JiraRestApi\JiraException","message":"CURL HTTP Request Failed: Status Code : 400, URL:https://sd-jira-staging.surf.nl/jira/rest/api/2/issue[\nError](https://sd-jira-staging.surf.nl/jira/rest/api/2/issue%5CnError) Message : {"errorMessages":[],"errors":{"fixVersions":"Field 'fixVersions' cannot be set. It is not on the appropriate screen, or unknown."}}","code":400,"file":"/var/www/html/vendor/lesstif/php-jira-rest-client/src/[JiraClient.ph](http://jiraclient.ph/)p:286"}},"level":500,"level_name":"CRITICAL","channel":"request","datetime":"2025-05-14T16:01:20.217118+02:00","extra":{}}

Deze komen hier voor:
> https://github.com/lesstif/php-jira-rest-client/issues/547
> https://github.com/lesstif/php-jira-rest-client/pull/548

--- a/composer.json
+++ b/composer.json
@@ -38,7 +38,7 @@
     "guzzlehttp/guzzle": "^7.9.2",
     "knplabs/knp-menu-bundle": "^3.5.0",
     "league/tactician-bundle": "^1.5.1",
-    "lesstif/php-jira-rest-client": "^5.9",
+    "lesstif/php-jira-rest-client": "dev-main#0b0d4f9cfcfb7104749c25c2b2da02a31ed786ff",
     "lexik/translation-bundle": "^v6.0",
     "nelmio/security-bundle": "^3.4.2",
     "openconext/monitor-bundle": "^4.3.1",
@@ -189,6 +189,12 @@
         "Fix dependency injection issue": "./patches/dependency-injection-for-TinymceBundle.patch"
       }
     },
+    "repositories": [
+      {
+        "type": "vcs",
+        "url": "https://github.com/lesstif/php-jira-rest-client.git"
+      }
+    ],
     "symfony": {
       "allow-contrib": false,
       "require": "6.4.*"

